financial-templates-lib/ExpiringMultiPartyEventClient.js

```diff
@@ -96,8 +96,7 @@ class ExpiringMultiPartyEventClient {
   };
 
   _update = async () => {
-    const currentBlockNumber = await web3.eth.getBlockNumber();
-
+    const currentBlockNumber = await this.web3.eth.getBlockNumber();
```

Comment:
Just curious, how was this working before?

Reply:
When run from a `truffle exec` it somehow worked but it should be scoped to use the `web3` object within the emp event client.